expo/src/customize/customize.css

```diff
@@ -0,0 +1,63 @@
+/*

```

Comment:
I love this, great work Japneet! This gives really fine-tune control for every color used, I really like it!
There seem to be a good number of duplicate hex codes around (e.g. #00C7C7, #FF7BAC, etc...). I have an idea for consolidating those: Let's make some coarse-grained variables that are easier to understand? I'm thinking variables names like `--primary`, `--secondary`, `--accent`, etc...that could be used within `:root`?

Reply:
(i.e. let's replace duplicated hex codes like #FF7BAC with `--primary` or whatever name best describes it, so that hackathon organizers only need to change that hex code once?)